examples/medplum-patient-intake-demo/src/bots/core/test-data/intake-form-test-data.ts

```diff
@@ -201,6 +201,178 @@ export const intakeResponse: QuestionnaireResponse = {
         },
       ],
     },
+    {
+      id: 'id-82',
```

Comment:
Is it okay to remove those `id`s from the test data, or is it a good pattern to have them?

Reply:
ok to remove!